src/endpoint/s3/s3_constants.js

```diff
@@ -10,3 +10,7 @@ const s3_const = exports;
 ///////////////
 
 s3_const.MAX_RULE_ID_LENGTH = 255;
+s3_const.STATUS_VAL = Object.freeze({
```

Comment:
I'm suggesting to rename it in case we will have more enum status in the future:
```suggestion
s3_const.LIFECYCLE_STATUS = Object.freeze({
```

Reply:
Make sense, Done.